### ISSUE TITLE
Fetch contributors image using GitHub API

### ISSUE THREAD

**Issue by iamrajiv** [wechaty/docusaurus#1451]
Instead of storing the contributor's images in the assets folder, it's better to fetch GitHub avatar images of contributors using GitHub API.

I have found these API:

```shell
https://avatars.githubusercontent.com/<username>
https://avatars.githubusercontent.com/u/<userid>
```

**Comment by huan** [comment]
Yes I agree that it's better to fetch GitHub avatar images for contributors!

I'd like to accept a PR to implement this great feature, thank you very much for the suggestion!
